packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/watcherAt.put..st

```diff
@@ -1,8 +1,11 @@
 accessing
-watcherAt: anInteger put: aBoolean
-	
+watcherAt: t1 put: t2 
```

Comment:
No clue what `t1` and `t2` are supposed to be... Maybe use a more meaningful name?

Reply:
Also good suggestion! :D